docs/antrea-ipam.md

```diff
@@ -193,18 +194,26 @@ restore its configurations at exit. Node may lose network connection when `antre
 or OVS daemons are stopped unexpectedly, which can be recovered by rebooting the Node.
 `AntreaIPAM` Pods' traffic will not be routed by local Node's network stack.
 
-All traffic to a local Pod will be sent to the Pod's OVS port directly, after the
-destination MAC is rewritten to the Pod's MAC address. This includes `AntreaIPAM` Pods
-and regular `Subnet per Node` IPAM Pods, even they are not in the same subnets.
-Inter-Node traffic will be sent to the Node network from the source Node, and forwarded
-to the destination Node by the Node network.
+Traffic from `AntreaIPAM` Pods without VLAN, regular `Subnet per Node` IPAM Pods, and K8s
+Nodes are recognized as VLAN 0 (untagged).
+
+All traffic to a local Pod in same VLAN will be sent to the Pod's OVS port directly,
+after the destination MAC is rewritten to the Pod's MAC address. This includes
```

Comment:
Do we always override destination MAC, or only for cross-subnet case? If the latter, probably remove this sentence. If you want can add it to the last sentence:

> This includes `AntreaIPAM` Pods and regular `Subnet per Node` IPAM Pods. If the source and destination are not in the same subnet, the packets' destination MAC will be overridden to the destination Pod's MAC".
same subnet.

Reply:
Yes. DstMAC will be rewritten always for same vlan